packages/react-link/src/components/Link/useLinkState.ts

```diff
@@ -14,10 +14,14 @@ export const useLinkState = (state: LinkState): LinkState => {
   if (state.root.as === 'a') {
     state.root.href = disabled ? undefined : state.root.href;
     state.root.tabIndex = disabled && !disabledFocusable ? undefined : 0;
+
+    // Add role="link" for disabled and disabledFocusable links.
+    if (disabled || disabledFocusable) {
+      state.root.role = role || 'link';
```

Comment:
when the DOM element is rendered as an anchor why do we need to add the same role to the element ?

Reply:
In talking with @smhigley she was telling me that a `Link` without an `href` (which we drop when the `Link` is disabled) is semantically treated as a `span`, which makes us setting `aria-disabled: true` moot when `disabled` or `disabledFocusable` are passed in. One way to give it the correct semantic meaning of a disabled `Link` to accessibility tools is to add `role="link"` when this happens.